tests/sles4sap/wmp_basic_test.pm

```diff
@@ -0,0 +1,75 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run Workload Memory Protection basic test
+# Maintainer: Alvaro Carvajal <acarvajal@suse.de>
+
+use base "sles4sap";
+use testapi;
+use utils qw(zypper_call file_content_replace);
+use version_utils qw(is_sle);
+use strict;
+use warnings;
+
+sub run {
+    my ($self)     = @_;
+    my $testname   = 'wmp_basic_tests';
+    my $logdir     = '/root/wmp_logs';
+    my $sapsvc     = '/usr/sap/sapservices';
+    my @testphases = qw(initial takeover takeback);
+
+    # WMP is a feature of SLES for SAP Applications 15+. Skip test in older systems
+    if (is_sle('<15')) {
+        record_info 'WMP', 'WMP is only available in SLES for SAP Applications 15+';
+        return;
+    }
+
+    # Download python test script in HOME directory only if it has not been downloaded before
+    if (script_run "ls -d /root/$testname") {
+        assert_script_run 'cd /root';
+        my $wmp_test_repo = "https://gitlab.suse.de/lpalovsky/$testname/-/archive/master/$testname-master.tar.gz";
```

Comment:
Maybe better to use a variable here, as in the future I think that this script could be hosted elsewhere and not in Lumir's repo, no?

Reply:
But it's already in a variable, or do you mean to define it in the first few lines of the function with the other variables?

Reply:
I mean an openQA variable.

Reply:
Done. Repo definition moved to a variable. Variable defined in the job schedules.

Let me know if you think it's better to have the variable defined elsewhere to remove it from the schedules.

Reply:
That's fine for me, and if we need to overwrite it we can either clone the job, change the YAML file or create a test suite description for it (temporarily of course!).